src/contoroller/account/account.test.js

```diff
@@ -0,0 +1,520 @@
+const AccountController = require("./account");
+
+describe("AccountController", () => {
+  describe("create", () => {
+    describe("when validateAccount throws an error", () => {
+      let req,
+        res,
+        next,
+        validationError,
+        mockValidateAccount,
+        mockResponseSend;
```

Comment:
Since you want to use these in multiple describe blocks, you can define these at the top level once and avoid repeating them over and over.